### PR TITLE
Fix some bugs in CLI docs

### PR DESCRIPTION
**Pull request by stjohnjohnson**
## Description

- Sort the commands alphabetically
- Add missing `json_keymap`
- Correct underscore to dash

## Types of Changes

- [x] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -125,6 +125,16 @@ This command examines your environment and alerts you to potential build or flas
 qmk doctor
 ```
 
+## `qmk json_keymap`
+
+Create a keymap.c from a QMK Configurator export.
```
Comment (fauxpark) [inline docs/cli.md]:
```suggestion
Creates a keymap.c from a QMK Configurator export.
```

File: docs/cli.md [hunk 2]
```diff
@@ -125,6 +125,16 @@ This command examines your environment and alerts you to potential build or flas
...
+
+**Usage**:
+
+```
+qmk json_keymap [-h] [-o OUTPUT] filename
```
Comment (fauxpark) [inline docs/cli.md]:
```suggestion
qmk json_keymap [-o OUTPUT] filename
```

**Comment by stjohnjohnson** [comment]
@fauxpark resolved and updated source to match

**Comment by drashna** [comment]
Looks like there is a merge conflict here.  Could you fix this?

**Comment by stjohnjohnson** [comment]
@drashna and @zvecr, updated :D

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**